core/src/test/scala/scalgorand/domain/asset/AssetSpec.scala

```diff
@@ -0,0 +1,37 @@
+package scalgorand.domain.asset
+
+import zio.Scope
+import zio.json.DecoderOps
+import zio.test.{Spec, TestEnvironment, ZIOSpecDefault, assertTrue}
+
+import scala.io.Source.fromFile
+
+object AssetSpec extends ZIOSpecDefault {
+  override def spec: Spec[TestEnvironment with Scope, Any] =
+    test("Test for Asset Serialization") {
+
+      val assetData =
```

Comment:
Can we refactor this into a data object as done in the example on GenesisData so that tests are reduced so something concise as shown below 
```Scala
    test("Test Genesis Serialisation"){
       val decoded = genesisData.fromJson[Genesis]
      assertTrue(decoded.isRight)
    }
```

Reply:
Adressed in the latest commit.